pkg/ddevapp/dotddev_assets/providers/pantheon.yaml

```diff
@@ -67,13 +67,16 @@ auth_command:
     terminus auth:login --machine-token="${TERMINUS_MACHINE_TOKEN}" || ( echo "terminus auth login failed, check your TERMINUS_MACHINE_TOKEN" && exit 1 )
     terminus aliases 2>/dev/null
 
+#make sure env is awake, and grab a fresh DB export
 db_pull_command:
   command: |
     # set -x   # You can enable bash debugging output by uncommenting
     set -eu -o pipefail
     echo "Using PANTHEON_SITE=${PANTHEON_SITE} PANTHEON_ENVIRONMENT=${PANTHEON_ENVIRONMENT}"
     pushd /var/www/html/.ddev/.downloads >/dev/null
-    terminus backup:get ${PANTHEON_SITE}.${PANTHEON_ENVIRONMENT} --element=db --to=db.sql.gz
+    terminus env:wake -- ${PANTHEON_SITE}.${PANTHEON_ENVIRONMENT}
+    FRESH_DB_DUMP_STRING=$(terminus connection:info "${PANTHEON_SITE}.${PANTHEON_ENVIRONMENT}" --field=mysql_command | sed 's,^mysql,mysqldump --no-autocommit --single-transaction --opt -Q,')
```

Comment:
Why is the sed in there?

Reply:
It comes from https://github.com/lando/pantheon/blob/58cf95d9849e5b7aae01e5830edbaeadc2d19c0a/scripts/pull.sh#L129

`terminus connection:info --field=mysql_command` receives a string like:

```
mysql -u ... -p ... -h ...
```

And `sed` replaces it, so the commands looks like this:

```
mysqldump --no-autocommit --single-transaction --opt -Q -u ... -p ... -h ...
```